src/simulation/monsters/bosses/Sarachnis.ts

```diff
@@ -0,0 +1,67 @@
+import LootTable from '../../../structures/LootTable';
+import SimpleMonster from '../../../structures/SimpleMonster';
+import RareSeedTable from '../../subtables/RareSeedTable';
+import NotedHerbTable from '../../subtables/NotedHerbTable';
+import { GemTable } from '../../subtables/RareDropTable';
+
+const SarachnisTable = new LootTable()
+	.oneIn(192, 'Dragon med helm')
+	.oneIn(384, 'Sarachnis cudgel')
+
+	/* Weapons and armour */
+	.add('Battlestaff', [8, 10], 2)
+	.add('Rune platebody', 1, 2)
+	.add('Rune med helm', 1, 2)
+	.add('Rune 2h sword', 1, 2)
+
+	/* Runes and ammunition */
+	.add('Blood rune', [80, 100], 5)
+	.add('Chaos rune', [175, 200], 5)
+	.add('Cosmic rune', [125, 150], 5)
+	.add('Death rune', [80, 100], 5)
+	.add('Soul rune', [80, 100], 5)
+	.add('Mithril arrow', [450, 600], 2)
+	.add('Mithril bolts', [175, 225], 2)
+
+	/* Seeds */
+	.add(RareSeedTable, 1, 2)
+
+	/* Herbs */
+	.add(NotedHerbTable, 1, 10)
```

Comment:
the herb table quoted on the wiki seems to be different than ones we already have defined in `subtables`  so you need to make that in this file as an internal loot table then call it here